test/integration/base.py

```diff
@@ -1,50 +1,46 @@
-import unittest
-
-from .pelikan_client import PelikanClient
-from .pelikan_server import PelikanServer
-
-SERVER_PORT = 12345
-ADMIN_PORT = 12346
+from client import DataClient, AdminClient
```

Comment:
I believe python's community prefers `.client` in this situation (and `client` will fail with python3) to prevent cases where names are ambiguous with top level modules.